packages/docusaurus-theme-classic/src/theme/CodeBlock/index.js

```diff
@@ -13,7 +13,29 @@ import Clipboard from 'clipboard';
 import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
 import styles from './styles.module.css';
 
-export default ({children, className: languageClassName}) => {
+const regexHighlightRange = /{([\d,-]+)}/;
```

Comment:
Can we use https://www.npmjs.com/package/parse-numeric-range instead for parsing? Allows for more options and might be more robust than custom regex.

Reply:
@yangshun done :heavy_check_mark: